.github/workflows/npm_publish_light.yml

```diff
@@ -0,0 +1,126 @@
+name: Build and Publish Cohort SDK NPMs
+on:
+    #workflow_call:
+    push:
+env:
+    SDK_JS_PACKAGE_NAME: "@kindredgroup/cohort_sdk_js"
+    SDK_CLIENT_PACKAGE_NAME: "@kindredgroup/cohort_sdk_client"
+jobs:
+    npm:
+        strategy:
+            fail-fast: false
+            matrix:
+                settings:
+                  - host: macos-latest
+                    dirSdkJs: packages/cohort_sdk_js
+                    dirSdkJsClient: cohort_sdk_client
+                    dirCohortInitiator: cohort_banking_initiator_js
+                    dirCohortReplicator: cohort_banking_replicator_js
+                    customiseHostCommand: echo "No customizations needed"
```

Comment:
I probably wouldn't pass custom commands over a variable and inject it. 

Two reasons:- 
- This feels a bit unsafe to me. 
- Also when I look at the actions it makes it harder to interpret the workflow when custom commands come from variables. (I referred to `npm.yml` file and that is why this comment)